modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -21,17 +21,29 @@ def self.for_user(user)
     end
 
     def get_appointments(type, start_date, end_date, pagination_params = {})
-      with_monitoring do
-        url = get_appointments_base_url(type)
+      params = date_params(start_date, end_date).merge(page_params(pagination_params)).merge(other_params).compact
 
-        response = perform(:get, url, params(start_date, end_date, pagination_params), headers(user))
+      with_monitoring do
+        response = perform(:get, get_appointments_base_url(type), params, headers(user))
         {
           data: deserialized_appointments(response.body, type),
           meta: pagination(pagination_params)
         }
       end
     end
 
+    def put_cancel_appointment(request_object_body)
+      params = VAOS::CancelForm.new(request_object_body).params
+      params.merge!(patient_identifier: { unique_id: user.icn, assigning_authority: 'ICN' })
+
+      with_monitoring do
+        perform(:put, put_appointment_url, params, headers(user))
+        ''
```

Comment:
Could we return `true` for success if there's no object to return?

Reply:
we could, but I figured we usually return JSON or raise an exception and this was at least closer to returning to JSON than boolean.

Reply:
the idea is that we want this controller action to return no response body -- the question here is what should the service return...

if we were passing around response objects with status and body, then that would be ideal. The general pattern has been to pass the response body. In this case there is none. So I'm thinking Nil or '' makes the most sense.